robocop/checkers/errors.py

```diff
@@ -22,55 +22,175 @@
         name="not-enough-whitespace-after-setting",
         msg="Provide at least two spaces after '{{ setting_name }}' setting",
         severity=RuleSeverity.ERROR,
+        docs="""
+        Example of rule violation::
+
+            *** Test Cases ***
+            Test
+                [Documentation] doc  # only one space after [Documentation]
+                Keyword
+                
+            *** Keywords ***
+            Keyword
+                [Documentation]  This is doc
+                [Arguments] ${var}  # only one space after [Arguments]
+                Should Be True  ${var}
+            
+        """,
     ),
     "0403": Rule(
         rule_id="0403",
         name="missing-keyword-name",
         msg="Missing keyword name when calling some values",
         severity=RuleSeverity.ERROR,
+        docs="""
+        Example of rule violation::
+        
+            *** Keywords ***
+            Keyword
+                ${var}
+                ${one}      ${two}
+
+        """,
     ),
     "0404": Rule(
         rule_id="0404",
         name="variables-import-with-args",
         msg="Robot and YAML variable files do not take arguments",
         severity=RuleSeverity.ERROR,
+        docs="""
+        Example of rule violation::
+        
+            *** Settings ***
+            Variables    vars.yaml    arg1
+            Variables    variables.robot    arg
+        
+        """,
     ),
     "0405": Rule(
         rule_id="0405",
         name="invalid-continuation-mark",
         msg="Invalid continuation mark '{{ mark }}'. It should be '...'",
         severity=RuleSeverity.ERROR,
+        docs="""
+        Example of rule violation::
+        
+            Keyword
+            ..  ${var}  # .. instead of ...
+            ...  1
+            ....  2  # .... instead of ...
+
+        """,
     ),
     # there is not-enough-whitespace-after-newline-marker for keyword calls already
     "0406": Rule(
         rule_id="0406",
         name="not-enough-whitespace-after-newline-marker",
         msg="Provide at least two spaces after '...' marker",
         severity=RuleSeverity.ERROR,
+        docs="""
+        Example of rule violation::
+        
+            @{LIST}  1
+            ... 2  # not enough whitespace
```

Comment:
Usually it's uncountable (like flour ;)) and it can mean one or more spaces/new lines etc in general:
https://www.wordhippo.com/what-is/the-plural-of/whitespace.html